crates/wire/Cargo.toml

```diff
@@ -8,6 +8,7 @@ bitcoin.workspace = true
 libp2p-identity.workspace = true
 musig2.workspace = true
 prost.workspace = true
+strata-p2p-db.path = "../db"
```

Comment:
I don't think `strata-p2p-wire` should import DB crate, let's instead create shared `types` crate for `OperatorPubkey`